site/content/extend/plugins/server/hello-world.md

```diff
@@ -27,6 +27,11 @@ Download the source code for the Mattermost server:
 go get -u github.com/mattermost/mattermost-server
 ```
 
+Define `$GOPATH`. By default, this is already `$HOME/go`, but it's helpful to make this explicit:
```

Comment:
Makes things a little less confusing if testing this out on a Docker container, for example, as root.